packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -86,14 +86,14 @@ function NetworkListbox({
     const { isArbitrum, isArbitrumNova } = isNetwork(value.chainID)
 
     if (!isArbitrum) {
-      return 'bg-[rgba(118,121,145,0.8)]'
+      return 'bg-[#454A75]'
     }
 
     if (isArbitrumNova) {
-      return 'bg-[rgba(255,206,162,0.8)]'
```

Comment:
<img width="1084" alt="image" src="https://user-images.githubusercontent.com/13184582/235169333-9884c8c4-8d6b-461a-93a2-76864fc2bc2a.png">
since nova orange has been updated, we should update it in `tailwind.config.ts` and use the var

btw Nova's button colour is wrong too

https://github.com/OffchainLabs/arbitrum-token-bridge/blob/75f485412716be388b779a2f32c611db9bc3e912/packages/arb-token-bridge-ui/tailwind.config.js#L54

<img width="237" alt="image" src="https://user-images.githubusercontent.com/13184582/235169673-332ecdc7-518d-402b-8c26-3898fa72e283.png">

Reply:
The final color for the `Move funds to...` button is the one currently implemented— the `darker/bg` variant of all the brand colors.

Reply:
Also updated the config with the correct color.